### DISCUSSION TITLE
Favourite repositories rearrange is impossible

### DISCUSSION THREAD

**Discussion by hemche** [community/community#142453] [Repositories]
From the Android mobile GitHub homepage, scroll down to the Favourite repositories section and edit.
It is impossible to drag repositories from top to bottom as it is triggering pull to refresh on the screen.

Fix would be to disable Pull to refresh on the edit favourite repositories screen.

**Comment by taki-on** [comment]
The issue is about the mobile version of GitHub.com. I will change the category.

BTW, you can re-order favorite repos in the GitHub Mobile for Android :)

**Reply by hemche** [reply]
I am seeing issue with Android github app only.

**Reply by hemche** [reply]
Rearranging the bottom one to top is possible. But, dragging projects on the top to down is not possible as it is triggering pull to refresh.
